e2e/nomostest/new.go

```diff
@@ -136,7 +136,7 @@ func NewOptStruct(testName, tmpDir string, t nomostesting.NTB, ntOptions ...ntop
 		optsStruct.RESTConfig.Burst = 75
 
 		// Disable client-side throttling for the test client, if server-side throttling is enabled.
```

Comment:
This comment needs to be updated to reflect `restconfig.UpdateQPS(optsStruct.RESTConfig, false)`.

Reply:
I see. Make sense.